probe/host/reporter.go

```diff
@@ -117,6 +117,17 @@ func (r *Reporter) Report() (report.Report, error) {
 	rep.Host = rep.Host.WithMetadataTemplates(MetadataTemplates)
 	rep.Host = rep.Host.WithMetricTemplates(MetricTemplates)
 
+
+	os, err := GetPrettyName()
```

Comment:
- This function could use a more explicit name, how about `GetOSName`?

- I would simplify the logic here: don't return a special `"unknown"` value but fallback to `runtime.GOOS` on any error:

```
os, err := GetOSName()
if err != nil {
        os = runtime.GOOS
}
```